### PR TITLE
Added click to mark phases as completed

### PR DESCRIPTION
**Pull request by okay-scam**
I'm not much of a dev but thought this feature would be handy for myself and others.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 const CLICKABLE_IMAGES_SELECTOR = '.phase-options img',
+	CLICKABLE_PHASES_SELECTOR = '.phases-completed img',
```
Comment (Nightfirecat) [inline script.js]: I'd prefer a new class of `.pattern` be applied to the pattern elements so it's more clear what that class means.

File: script.js [hunk 2]
```diff
@@ -85,3 +92,15 @@ function showNode(node) {
 		node.setAttribute('style', VISIBLE_ELEMENT_STYLE);
 	}
 }
+
+function toggleCompleted(event) {
+	if (event.srcElement.className !== "completed") {
+		event.srcElement.className = (event.srcElement.className == "toggle-completed") ? "" : "toggle-completed";
```
Comment (Nightfirecat) [inline script.js]: All of the class checking should be handled using [`Element.classList`](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList) (namely `classList#contains()` and `classList#toggle()`), like is being done in `resetToggleCompleted()`.

File: script.js [hunk 3]
```diff
@@ -85,3 +92,15 @@ function showNode(node) {
...
+	}
+}
+
+function resetToggleCompleted() {
+	for (const node of document.querySelectorAll(CLICKABLE_PHASES_SELECTOR)) {
+		node.classList.remove('toggle-completed');
+	}
+}
```
Comment (Nightfirecat) [inline script.js]: Add newline at EOF
```suggestion
}

```

**Review by Nightfirecat — CHANGES_REQUESTED** [review]

File: script.js (hunk 1 above)
Comment (okay-scam) [reply script.js]: Do you mean just to rename the class to eg. .pattern-completed?

Comment (Nightfirecat) [reply script.js]: Changing the newly-added class name from `.phases-completed` to `.pattern` is what I mean.

**Comment by okay-scam** [comment]
Pushed the requested changes

**Comment by Nightfirecat** [comment]
I touched up a couple things and added this in ac37d7dff5a20d520fd95250446aab7c08a42612. Thanks for your contribution!

**Pull request closed without merging**